scripts/tools/fetch-pull-request

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ $# -eq 0 ]
+    then
+        echo "Id of pull request not supplied"
```

Comment:
should i keep the echo if we are keeping the set -x then we will have messages like 

`+ '[' 0 -eq 0 ']'
+ echo 'Pull request id not given'
Pull request id not given`